### PR TITLE
Declare py-beautifulsoup4's dependency on py-setuptools

### PR DESCRIPTION
**Pull request by eschnett**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-beautifulsoup4/package.py [hunk 1]
```diff
@@ -37,5 +37,7 @@ class PyBeautifulsoup4(Package):
 
     extends('python')
 
+    depends_on('py-setuptools')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-beautifulsoup4/package.py]: this should probably be a `type='build'` dependency

Comment (eschnett) [inline var/spack/repos/builtin/packages/py-beautifulsoup4/package.py]: Yes. Fixed.

**Comment by adamjstewart** [comment]
LGTM

**Comment by eschnett** [comment]
Ping.

**Pull request merged**